eslzArm/managementGroupTemplates/policyDefinitions/gov/fairfaxPolicies.json

```diff
@@ -272,10 +291,13 @@
                     "/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c"
                   ],
                   "deployment": {
-                    "location": "usgovvirginia",
+                    "location": "northeurope",
```

Comment:
Can you pls update the location to be 'usgovvirginia' and align with the other DINE policies targeting Microsoft.Resources/subscriptions in this file?

Reply:
Good spot. Done 👍